### DISCUSSION TITLE
Cool arrow idea

### DISCUSSION THREAD

**Discussion by Randomllyin** [Leather128/LeathersFunkinEngine#240] [Ideas]
Hear me out
What if
You could set BF to have 4 keys
And then give the opponent 16 notes
and then put 8 of the opponent notes on the bottom
Kinda like this (Default arrows are BF)
![image](https://user-images.githubusercontent.com/37484220/167317865-88d29bf3-3fab-411a-b6d5-97b960bd423b.png)
I put BF's arrows on the right like that because the original mockup was gonna have the arrows going right

**Comment by redstonemaster60** [comment]
honestly you could probably just do that with modcharting if you were good enough at it but it would be tough
